crawl-ref/source/spl-util.cc

```diff
@@ -1649,11 +1648,16 @@ bool spell_no_hostile_in_range(spell_type spell)
         return false;
 
     case SPELL_OLGREBS_TOXIC_RADIANCE:
+        return cast_toxic_radiance(&you, pow, false, true) == spret::abort;
```

Comment:
This OTR case could perhaps be moved lower in the function, for a visual grouping with the other spells returning a `cast_spell_name` handler?

Reply:
Good idea, done